DFE-Digital/curriculum-materials
# Adds previews for open office documents

### Context
So teachers can preview open document format documents inline in the browser, we need to generate previews for them in a suitable format.

### Changes proposed in this pull request
Adds pdf previews for open document files.
Adds a script to generate previews.
Updates the allowed content type for slide deck previews from png to pdf.

### Guidance to review
Manual testing if you're so inclined:
re seed your database and preview an open document resource. Expect it to preview inline.

Testing the script.
Remove previews `find db/seeds/data | grep ".preview" | xargs rm`
Run the script if you have unocov installed `db/seeds/preview-generator.sh`. Unocov is a bit flaky on my mac so you may need to run it a few times.
Alternatively running the script via docker seems to work fine first go
`docker run -v "$(pwd):/tmp/app" --rm -it skylarkelty/docker-unoconv /tmp/app/db/seeds/preview-generator.sh`
